src/domains/transaction/pages/SendVote/SendVote.tsx

```diff
@@ -135,8 +184,9 @@ export const SendVote = () => {
 
 			appendParameters(parameters, "vote", voteDelegates);
 
+			//	TODO: redirect to general votes page if wallet is not defined.
```

Comment:
🤔

Reply:
@ItsANameToo fixed. Redirecting to dashboard if the user comes from a deeplink url. Thanks!